### PR TITLE
Add code stats

### PR DESCRIPTION
**Pull request by danielaRiesgo**
## Summary ##
**Trello card**: https://trello.com/c/5pYvh9NN/12-configurar-codestats

Adding code stats for code coverage.

### CODE REVIEW

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:x: Code Coverage: 54.59 - Minimum: 70.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-core-ios/branches/add-code-stats) for more details.

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:x: Code Coverage: 54.59 - Minimum: 70.0
:x: Slather: 54.59 - Minimum: 70.0
:white_check_mark: Swiftlint: 98.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-core-ios/branches/add-code-stats) for more details.

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:x: Code Coverage: 54.59 - Minimum: 70.0
:x: Slather: 54.59 - Minimum: 70.0
:white_check_mark: Swiftlint: 98.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-core-ios/branches/add-code-stats) for more details.

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:x: Code Coverage: 54.59 - Minimum: 70.0
:x: Slather: 54.59 - Minimum: 70.0
:white_check_mark: Swiftlint: 98.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-core-ios/branches/add-code-stats) for more details.

File: .codestats.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+   ci: 'BITRISE'
+   url: http://codestats.wolox.com.ar/
+   token: <%= ENV['CODE_STATS_TOKEN'] %>
+metrics:
+   slather:
+     enabled: true
+     minimum: 70
+     upload_report: false
+   swiftlint:
+       enabled: true
```
Comment (pablog) [inline .codestats.yml]: This is not properly indented.

File: Gemfile [hunk 2]
```diff
@@ -3,6 +3,5 @@ source 'https://rubygems.org'
 gem 'fastlane'
 gem 'xcpretty'
 gem 'carthage_cache'
-gem 'synx'
-gem 'linterbot'
 gem 'slather'
+gem 'codestats-metrics-reporter', :github => 'Wolox/codestats-metrics-reporter', :branch => 'swiftlint'
```
Comment (pablog) [inline Gemfile]: You can use `gem 'codestats-metrics-reporter'`. Branch `swiftlint` is already merged. Make sure you're using `0.1.13`

File: .codestats.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+config:
+   ci: 'BITRISE'
+   url: http://codestats.wolox.com.ar/
+   token: <%= ENV['CODE_STATS_TOKEN'] %>
+metrics:
+   slather:
+     enabled: true
+     minimum: 70
+     upload_report: true
```
Comment (pablog) [inline .codestats.yml]: I'd leave it disabled. For the report to be uploaded you need to configure some keys in CI, but we'll do that once we have a workaround for the secrets in PRs.

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:x: Code Coverage: 54.59 - Minimum: 70.0
:x: Code Coverage (Slather): 54.59 - Minimum: 70.0
:white_check_mark: Code Quality (Swiftlint): 98.0
:x: Slather: 54.59 - Minimum: 70.0
:white_check_mark: Swiftlint: 98.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-core-ios/branches/add-code-stats) for more details.

**Pull request merged**